docs/source/getting-started/first-feature.rst

```diff
@@ -78,11 +78,6 @@ Open a new terminal and run the following commands::
 
 The result is an up-to-date TypeScript interface to our DAML model, in particular to the new ``Message`` template and ``SendMessage`` choice.
 
-To make sure that Yarn picks up the newly generated JavaScript code,
-we have to run the following command in the ``ui`` directory::
-
-  npm install --frozen-lockfile
-
```

Comment:
Should we mention that `npm start` will pick up the changes automatically or would that only cause confusion for users who didn't even think about this problem in the first place?

Reply:
I'd say deciding that goes a bit beyond the quickfix on a patched release, unless someone else has a strong opinion either way.

Reply:
(And can express it right now.)

Reply:
Let's skip it then. I guess it wouldn't be too helpful anyway.